lib/logger.js

```diff
@@ -0,0 +1,24 @@
+const { createLogger, format, transports } = require('winston');
+
+const httpTransportOptions = (datadogToken) => {
+	return {
+		host: 'http-intake.logs.datadoghq.com',
+		path: `/api/v2/logs?dd-api-key=${datadogToken}&ddsource=nodejs&service=connectors`,
+		ssl: true
+	}; 
+};
+
+
+const initialiseLogger = (datadogToken) => {
+	const transportsList = [process.env.NODE_ENV === 'production' ? new transports.Http(httpTransportOptions(datadogToken)) : new transports.Console()];
```

Comment:
Is `NODE_ENV` always? I've only ever seen it set during development.

If this logger should only ever run when the code is running as a lambda function, then it might be better to check for `AWS_LAMBDA_FUNCTION_NAME` instead, as AWS provides this env for all lambda runtimes: 
https://docs.aws.amazon.com/lambda/latest/dg/configuration-envvars.html#configuration-envvars-runtime